mart/configs/model/detection/preprocessor_multi_modal.yaml

```diff
@@ -0,0 +1,12 @@
+# @package model.modules.preprocessor
```

Comment:
Why not create a modules directory? Why does this live under detection when it has nothing to do with detection?

Reply:
model/detection -> model/modules